corehq/apps/integration/static/integration/js/dialer/dialer_utils.js

```diff
@@ -1,296 +1,296 @@
-/**

```

Comment:
It looks like all the js changes are whitespace, plus the eslint tag. Can you just remove them? It'll save your name from coming up when future devs blame this file.